server/plugin.go

```diff
@@ -20,6 +27,15 @@ const (
 	botUserName    = "zoom"
 	botDisplayName = "Zoom"
 	botDescription = "Created by the Zoom plugin."
+
+	zoomDefaultUrl    = "https://zoom.us"
+	zoomDefaultAPIUrl = "https://api.zoom.com/v2"
+	zoomTokenKey      = "_zoomtoken"
+	zoomEmailKey      = "_zoomemail"
+
+	zoomStateLength   = 3
+	zoomOAuthmessage  = "[Click here to link your Zoom account.](%s/plugins/zoom/oauth/connect?channelID=%s)"
+	zoomEmailMismatch = "We could not verify your Mattermost account in Zoom. Please ensure that your Mattermost email address matches your Zoom login email address."
```

Comment:
@hanzei Could we echo out the user's email that we are using for auth so they can easily see if it doesn't match?  

```suggestion
	zoomEmailMismatch = "We could not verify your Mattermost account in Zoom. Please ensure that your Mattermost email address ({{EMAIL_ADDRESS}}) matches your Zoom login email address."
```

Reply:
@hanzei @aaronrothschild I have re-formatted this message to include email, instead of handling the `{{EMAIL_ADDRESS}}`, server includes the email as part of the error message.